api/src/opentrons/protocol_engine/commands/calibration/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""Calibration Module protocol commands."""
```

Comment:
```suggestion
"""Calibration commands."""
```

I wouldn't mention "module" here because it might be confused with 

I would also just say "commands" instead of "protocol commands." They *are* protocol commands in the sense that they're just like any other command in Protocol Engine, but they're not supposed to appear in an actual protocol, so... 🤷